### DISCUSSION TITLE
Make "default sort order" a section setting

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9421] [Ideas]
> Created by: Timothy Kelty (timkelty@gmail.com) on 2015/04/02 12:33:05 +0000
> Votes at time of UserVoice import: 3

-----------------------------------

Would be nice if you could specify the default sort order for a section in the cp when you create it ("title asc", etc)

**Comment by joshuabaker** [comment]
Had a client request this today, specific to asset selection from a field. Would this apply there also?

**Comment by kringkaste** [comment]
This would be very helpful. We also had a client request for this.
